lte/protos/BUILD.bazel

```diff
@@ -354,6 +354,16 @@ cpp_grpc_library(
     deps = [":sms_orc8r_cpp_proto"],
 )
 
+proto_library(
```

Comment:
This comment is actually for line 367:
`diam_errors_proto` needs to be added as a dependency, i.e., after line 369 add
`deps = [":diam_errors_proto"],`

can be tested on vm or dev container via `bazel build lte/protos:subscriberauth_proto`. Or build all proto libraries via `bazel build lte/protos/...`.

Reply:
It doesn't seem to be enough, I still see build errors.
I am not sure how to fix the build since I cannot run VirtualBox on M1 Mac & don't see relevant magma bazel build instructions...

Reply:
Hi, the Bazel build seems to be working again [1]. Thank you for using the suggested fixes :) The other failing Jobs seem to be not related to this issue - the Bazel-ification is an ongoing effort, i.e., required CI builds are not affected yet. Feel free to join #bazel in the Magma slack if you have more questions.

[1] https://github.com/magma/magma/actions/runs/1896125985